documentation/shared/attributes.adoc

```diff
@@ -127,7 +127,6 @@
 :KafkaTopicApiVersion: kafka.strimzi.io/v1beta1
 :KafkaUserApiVersion: kafka.strimzi.io/v1beta1
 :KafkaMirrorMakerApiVersion: kafka.strimzi.io/v1beta1
-:ApiVersion: v1beta1
```

Comment:
Quick search suggests that this is used in a lot more files than just this one ... are you sure it can be safely removed?

Reply:
It was a duplicate. There's also one on line 123.